apps/nextjs-website/src/components/molecules/PartRenderer/PartRenderer.tsx

```diff
@@ -20,6 +21,8 @@ const PartRenderer = ({ part }: PartRendererProps): ReactNode | null => {
       return <CodeBlockPart {...part} />;
     case 'innerHTML':
       return <InnerHtmlPart html={part.html} />;
+    case 'innerHTMLLazyLoaded':
```

Comment:
What are the differences between `InnerHtmlPart` and `InnertHtmlLazyLoadedPart` components?

Reply:
👍 Does make sense keep just `InnertHtmlLazyLoadedPart` instead both version?